chsdi/lib/opentransapi/opentransapi.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+import requests
+import xml.etree.ElementTree as et
+from pytz import timezone
+from datetime import datetime, timedelta
+from pyramid.threadlocal import get_current_registry
+
+
+class OpenTrans:
+
+    def __init__(self):
+        self.open_trans_api_key = get_current_registry().settings['opentrans_api_key']  # Get API key from config .ini
```

Comment:
not super important but it would make more sense to do that in the view and pass the api key as argument here.